### DISCUSSION TITLE
Google calendar ical email for the new event

### DISCUSSION THREAD

**Discussion by angelocioci1985** [derekantrican/GAS-ICS-Sync#389] [Q&A]
Hi at all, I use this script to refresh every minute the calendar https://gist.github.com/gene1wood/02ed0d36f62d791518e452f55344240d
Ant everythings works fine, now my problem is, how can I receive/send a mail every time there is a new event?

Many thanks

Angelo

**Comment by jonas0b1011001** [comment]
Does https://github.com/derekantrican/GAS-ICS-Sync/blob/9990d1b037bbc3e2d72f130f8a815c0f71bd3bcd/Code.gs#L50 not suit your needs?

**Comment by angelocioci1985** [comment]
Hi Jonas, no I don't receive any email, why?

Il giorno mar 12 dic 2023 alle ore 02:44 jonas0b1011001 <
***@***.***> ha scritto:

**Comment by jonas0b1011001** [comment]
Have you specified a valid mail address just below that setting?

**Comment by angelocioci1985** [comment]
emailSummary set to true or to false?

Il giorno mar 12 dic 2023 alle ore 02:51 Angelo Cioci <
***@***.***> ha scritto:

**Reply by jonas0b1011001** [reply]
```
var emailSummary = true;
var email = ""; <- add your mail address
```

**Comment by angelocioci1985** [comment]
I set a valid e-mail address, I also try another one Inviato da iPhoneIl giorno 11 dic 2023, alle ore 19:49, jonas0b1011001 ***@***.***> ha scritto:﻿
Have you specified a valid mail address just below that setting?

**Comment by jonas0b1011001** [comment]
1. What version of the script are you running? You find your version in [Helpers.gs](https://github.com/derekantrican/GAS-ICS-Sync/blob/master/Helpers.gs#L991)
2. Is the script executing without errors? You can check the logs at [https://script.google.com/home/executions](https://script.google.com/home/executions)

**Comment by angelocioci1985** [comment]
Version 5.7Inviato da iPhoneIl giorno 11 dic 2023, alle ore 20:12, jonas0b1011001 ***@***.***> ha scritto:﻿

What version of the script are you running? You find your version in Helpers.gs
Is the script executing without errors? You can check the logs at https://script.google.com/home/executions

**Comment by angelocioci1985** [comment]
Version 5.7Error:DNS error: http://icsUrl2 at callWithBackoff(Helpers:939:9) at fetchSourceCalendars(Helpers:80:5) at startSync(Code:159:21)Inviato da iPhoneIl giorno 11 dic 2023, alle ore 20:12, jonas0b1011001 ***@***.***> ha scritto:﻿

What version of the script are you running? You find your version in Helpers.gs
Is the script executing without errors? You can check the logs at https://script.google.com/home/executions

**Comment by jonas0b1011001** [comment]
Your `var sourceCalendars` is malformatted, remove all unchanged example entries
```
,
  ["icsUrl2", "targetCalendar2"],
  ["icsUrl3", "targetCalendar1"]
```
so you end up with something like
```
var sourceCalendars = [

  ["https://p24-calendars.icloud.com/holidays/us_en.ics", "US Holidays"]

];
```

**Comment by angelocioci1985** [comment]
I change the calendar and the email but this is my script

var sourceCalendars = [                // The ics/ical urls that you want
to get events from along with their target calendars (list a new row for
each mapping of ICS url to Google Calendar)
                                       // For instance: ["
https://p24-calendars.icloud.com/holidays/us_en.ics", "US Holidays"]
                                       // Or with colors following mapping
https://developers.google.com/apps-script/reference/calendar/event-color,
                                       // for instance: ["
https://p24-calendars.icloud.com/holidays/us_en.ics", "US Holidays", "11"]
  ["
https://s.sync.rentman.eu/ical.php?c=cp&i=33&t=b4YmTgSb5W6%2Bo6IcAZun2NNqHDcCn3PpY2Xq04%3D%3Aìkijdf635859d062234b4a8fc0f768%3A%3A%3Abdd558fd204af41cabd8b29abc29a7a9#1
<https://s.sync.rentman.eu/ical.php?c=cp&i=33&t=b4YmTgSb5W6%2Bo6IcAZun2NNqHDcCn3PpY2Xq04%3D%3A%C3%ACkijdf635859d062234b4a8fc0f768%3A%3A%3Abdd558fd204af41cabd8b29abc29a7a9#1>",
"RENTMAN5", "11"]

];

var howFrequent = 1;                  // What interval (minutes) to run
this script on to check for new events
var onlyFutureEvents = false;          // If you turn this to "true", past
events will not be synced (this will also removed past events from the
target calendar if removeEventsFromCalendar is true)
var addEventsToCalendar = true;        // If you turn this to "false", you
can check the log (View > Logs) to make sure your events are being read
correctly before turning this on
var modifyExistingEvents = true;       // If you turn this to "false", any
event in the feed that was modified after being added to the calendar will
not update
var removeEventsFromCalendar = true;   // If you turn this to "true", any
event created by the script that is not found in the feed will be removed.
var addAlerts = "yes";                 // Whether to add the ics/ical
alerts as notifications on the Google Calendar events or revert to the
calendar's default reminders ("yes", "no", "default").
var addOrganizerToTitle = false;       // Whether to prefix the event name
with the event organiser for further clarity
var descriptionAsTitles = false;       // Whether to use the ics/ical
descriptions as titles (true) or to use the normal titles as titles (false)
var addCalToTitle = false;             // Whether to add the source
calendar to title
var addAttendees = false;              // Whether to add the attendee list.
If true, duplicate events will be automatically added to the attendees'
calendar.
var defaultAllDayReminder = -1;        // Default reminder for all day
events in minutes before the day of the event (-1 = no reminder, the value
has to be between 0 and 40320)
                                       // See
https://github.com/derekantrican/GAS-ICS-Sync/issues/75 for why this is
neccessary.
var overrideVisibility = "public";           // Changes the visibility of
the event ("default", "public", "private", "confidential"). Anything else
will revert to the class value of the ICAL event.
var addTasks = false;

var emailSummary = true;              // Will email you when an event is
added/modified/removed to your calendar
var email = ***@***.***";                        // OPTIONAL: If
"emailSummary" is set to true or you want to receive update notifications,
you will need to provide your email address

/*

Il giorno mar 12 dic 2023 alle ore 06:34 jonas0b1011001 <
***@***.***> ha scritto:

**Reply by jonas0b1011001** [reply]
And the log still shows the same error as before?

`var email = ***@***.***";`
Is missing the leading **"**

The link you are trying to sync does not work for me, you have to recheck if it's valid.

**Comment by jonas0b1011001** [comment]
I assume this is resolved / no longer relevant, Closing.
